# Greening lab database
This is the repository for the metabolic marker gene databases compiled by lab members of Greening lab (http://www.greeninglab.com/) since 2016. 

The original database was released in 2020 on Figshare yet no longer accessible (https://bridges.monash.edu/collections/Greening_lab_metabolic_marker_gene_databases/5230745). 

The extended and modified database are migrated to this github page.

## Original database (2020)
**Carbon fixation**<br />
AclB	ATP-citrate lyase (rTCA cycle)<br />
AcsB	Acetyl-CoA synthase (Wood−Ljungdahl pathway)<br />
HbsC	Crenarchaeota 4-hydroxybutyryl-CoA synthase (4-hydroxybutyrate cycle)<br />
HbsT	Thaumarchaeota  4-hydroxybutyryl-CoA synthase (4-hydroxybutyrate cycle)<br />
Mcr	  Malonyl-CoA reductase (3-hydroxypropionate cycle)<br />
RbcL	RuBisCO (CBB cycle; also include form II & III RuBisCO)<br />

**Trace gas metabolism**<br />
CooS	  Anaerobic [NiFe]-carbon monoxide dehydrogenase (anaerobic CO oxidation)<br />
CoxL	  Form I [MoCu]-carbon monoxide dehydrogenase (aerobic CO oxidation)<br />
[Fe]	  [Fe]-hydrogenase (H2 metabolism)<br />
[FeFe]  [FeFe]-hydrogenase (H2 metabolism)<br />
[NiFe]  [NiFe]-hydrogenase (H2 metabolism)<br />
McrA	  Methyl-CoM reductase (methanogenesis) [updated in 2024]<br />
rMcrA	  Reverse methyl-CoM reductase (anaerobic CH4 oxidation) [updated in 2024]<br />
PmoA	  Particulate methane monooxygenase (aerobic CH4 oxidation) [updated in 2024]<br />
MmoA    Soluble methane monooxygenase (aerobic CH4 oxidation) [updated in 2024]<br />
IsoA		Isoprene monooxygenase (isoprene oxidation) [updated in 2024]<br />

**Nitrogen metabolism**<br />
AmoA	  Ammonia monooxygenase (ammonia oxidation) [updated in 2024]<br />
HzsA	  Hydrazine synthase (anammox)<br />
NxrA	  Nitrite oxidoreductase (nitrite oxidation)<br />
NifH	  Nitrogenase (nitrogen fixation)<br />
NapA    Periplasmic nitrate reductase (nitrate reduction)<br />
NarG	  Dissimilatory nitrate reductase (nitrate reduction)<br />
NrfA	  Ammonifying nitrite reductase (nitrite reduction to ammonium)<br />
NirK    Copper-containing nitrite reductase (nitrite reduction to nitric oxide)<br />
NirS	  Cytochrome bd1 nitrite reductase (nitrite reduction to nitric oxide)<br />
NorB	  Cytochrome c dependent nitric oxide reductase (nitric oxide reduction)<br />
NosZ	  Nitrous oxide reductase (N2O reduction)<br />
Nod	    Nitric oxide dismutase (oxygenation via nitric oxide disproportionation)<br />

**Sulfur metabolism**<br />
FCC     Flavocytochrome c sulfide dehydrogenase (sulfide oxidation)<br />
Sqr	    Sulfide quinone oxidoreductase (sulfide oxidation)<br />
SoxB	  Thiosulfohydrolase (thiosulfate oxidation)<br />
rDsrA	  Oxidative Dsr (sulfite oxidation)<br />
AsrA	  Anaerobic sulfite reductase (sulfate reduction)<br />
DsrA	  Dissimilatory sulfite reductase (sulfate reduction)<br />
Sor	    Sulfur oxygenase reductase (sulfur disproportionation)<br />

**Alternative electron donor**<br />
ARO	  Arsenite oxidase (arsenite oxidation)<br />
Cyc2	Iron oxidising cytochrome (iron oxidation)<br />
FdhA	Formate dehydrogenase (formate oxidation)<br />

**Alternative electron acceptor**<br />
FrdA	Fumarate reductase (fumarate reduction)<br />
MtrB	Decaheme iron reductase (iron reduction)<br />
OmcB	Polyheme iron reductase (iron reduction)<br />
RdhA	Reductive dehalogenase (reductive dehalogenation)<br />
YgfK	Selenate reductase  (selenate reduction)<br />

**Phototrophy**<br />
PsaA	Photosystem 1 (photosystem 1 dependent phototrophy)<br />
PsbA	Photosystem 2 (photosystem 2 dependent phototrophy)<br />
RHO	  Microbial rhodopsin (rhodopsin dependent phototrophy)<br />

**Aerobic respiration**<br />
AtpA	ATP synthase (ATP synthesis)<br />
SdhA	Succinate dehydrogenase (succinate oxidation)<br />
CoxA	Cytochrome aa3 oxidase (aerobic respiration)<br />
CyoA	Cytochrome bo3 oxidase (aerobic respiration)<br />
CcoN	Cytochrome cbb3 oxidase (aerobic respiration)<br />
CydA	Cytochrome bd oxidase (aerobic respiration)<br />
NuoF	Complex I F subunit (NADH oxidation)<br />

**Other**<br />
ArsC	Arsenate reductase (arsenic detoxification)<br />

## Extended and modified database (2024)
**Trace gas metabolism**<br />
McrA	  Methyl-CoM reductase (methanogenesis) [updated in 2024]<br />
rMcrA	  Reverse methyl-CoM reductase (anaerobic CH4 oxidation) [updated in 2024]<br />
PmoA	  Particulate methane monooxygenase (aerobic CH4 oxidation) [updated in 2024]<br />
MmoX    Soluble methane monooxygenase  (Group III SDIMO) (aerobic CH4 oxidation) [updated in 2024]<br />

**Volatile organic compound metabolism**<br />
AcrA	Alkyl-CoM reductase (anaerobic alkane oxidation)<br />
PxmA	PmoA-like monooxygenase (putative aerobic alkane oxidation)<br />
HmoA	Particulate hydrocarbon monooxygenase (aerobic hydrocarbon oxidation)<br />
TmoA	Toluene monooxygenase (Group I SDIMO) (toluene oxidation)<br />
IsoA	Isoprene monooxygenase (Group I SDIMO) (isoprene oxidation) [updated in 2024]<br />
DmpN	Phenol monooxygenase (Group II SDIMO) (phenol oxidation)<br />
BmoX	Soluble short-chain alkane monooxygenase (Group III SDIMO) (aerobic short-chain alkane oxidation)<br />
EtnC	Soluble alkene monooxygenase (Group IV SDIMO) (aerobic alkene oxidation)<br />
PrmA	Soluble propane monooxygenase (Group V & VI SDIMO) (aerobic propane oxidation)<br />
ZmoA	Soluble alkene-active monooxygenase (Group VII SDIMO) (putative aerobic alkene oxidation)<br />
SmoX	Unclassified soluble diiron monooxygenase<br />

**Nitrogen metabolism**<br />
AmoA	  Ammonia monooxygenase (ammonia oxidation) [updated in 2024]<br />

# Citation
**Original 2020 database**<br />
1. Greening C†\* & Geier R†, Wang C, Woods LC, Morales SE, McDonald MJ, Rushton-Green R, Morgan XC, Koike S, Leahy SC, Kelly WJ, Cann I, Attwood GT, Cook GM, Mackie RI* (2019). Diverse hydrogen production and consumption pathways influence methane production in ruminants. The ISME Journal 13, 2617-2632<br />
2. Ortiz M† & Leung PM†\*, Shelley G, Jirapanjawat T, Nauer PA, Van Goethem MW, Bay SK, Islam ZF, Jordaan K, Vikram S, Chown SL, Hogg ID, Makhalanyane TP, Grinter R, Cowan DA\*, Greening C\* (2021). Multiple energy sources and metabolic strategies sustain microbial diversity in Antarctic desert soils. PNAS 118, e2025322118<br />

**Extended and modified 2024 database**<br />
1. Leung PM†\* & Jeffrey LC†\*, Bay SK, Gomez-Alvarez P, Hall M, Johnston SG, Dittmann J, Jirapanjawat T, Hutchinson TF, Coleman NV, Dong X, Deschaseaux E, Maher DT & Greening C\* (2025). Bark microbiota modulate climate-active gas fluxes in Australian forests. Science 391(6781), 149<br />

**[NiFe]-, [FeFe]-, and [Fe]-hydrogenases**<br />
1. Søndergaard D\*, Pedersen CNS, Greening C\* (2016) HydDB: a web tool for hydrogenase classification and analysis. Scientific Reports 6, 34212<br />
2. Greening C\*, Biswas A, Carere CR, Jackson CJ, Taylor MC, Stott MB, Cook GM, Morales SE\* (2016). Genomic and metagenomic surveys of hydrogenase distribution indicate H2 is a widely-utilised energy source for microbial growth and survival. The ISME Journal 10, 761-777<br />

**Form I carbon monoxide dehydrogenase**<br />
1. Cordero PRF† & Bayly K†, Leung PM, Huang C, Islam ZF, Schittenhelm R, King GM, Greening C\* (2019). Atmospheric carbon monoxide oxidation is a widespread mechanism supporting microbial survival. The ISME Journal 13, 2868-2881<br />
